### PR TITLE
close parentheses, make link to guide more obvious

### PR DESCRIPTION
**Pull request by dholbach**
- close parentheses
- make link to guide more obvious

### CODE REVIEW

**Review by stefanprodan — APPROVED** [review]

File: docs/index.md [hunk 1]
```diff
@@ -50,13 +50,16 @@ Components:
     - [Alert CRD](components/notification/alert.md)
     - [Receiver CRD](components/notification/receiver.md)
 
-To get started with the toolkit please follow this [guide](get-started/index.md).
+## Get Started
+
+!!!hint "Get started with the GitOps Toolkit!"
```
Comment (hiddeco) [inline docs/index.md:55]: Given this already renders a title, do we need the `# Get Started`?

Comment (dholbach) [reply docs/index.md:55]: Screenshot from 2020-10-06 14-43-41

Comment (dholbach) [reply docs/index.md:55]: We could point to some individual guides in that section further down the line?

Comment (hiddeco) [reply docs/index.md:55]: Oh, this looks great as is.

**Review by hiddeco — APPROVED** [review]

**Pull request merged**